hetu-docs/en/admin/properties.md

```diff
@@ -922,3 +922,64 @@ helps with cache affinity scheduling.
 > After the configured time elapsed and no response received, then client connection consider that to be failure in submission of request.
 >
 > (Note: this parameter should be configured with higher time when in high load environment)
+
+## Gossip Protocol Configurations for Failure Detection
```

Comment:
All thse should be under the Failure recovery configuration profile

Reply:
ok